web/app/js/components/ServiceMesh.jsx

```diff
@@ -178,10 +179,7 @@ export default class ServiceMesh extends React.Component {
     return _(componentNames)
       .map((name, id) => {
         let componentPods = _.get(podIndex, _.get(componentDeploys, id), []);
-        let podStatuses = _.map(componentPods, p => {
-          return { name: p.name, value: p.status === "Running" ? "good" : "bad" };
-        });
-        return { name: name, pods: _.sortBy(podStatuses, "name") };
+        return { name: name, pods: _.sortBy(getComponentPods(componentPods), "name") };
```

Comment:
if `getComponentPods()` is already doing a `.sortBy("name")`, is the `_.sortBy(` here necessary?

Reply:
You're right @siggy. Thanks for catching it. After removing it, the deployment names are still sorted 🙌